pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -654,6 +654,16 @@ func DefaultOpenAPIConfig(config configapi.MasterConfig) *openapicommon.Config {
 			},
 		}
 	}
+	if config.OAuthConfig != nil {
+		securityDefinitions["Oauth2AccessToken"] = &spec.SecurityScheme{
+			SecuritySchemeProps: spec.SecuritySchemeProps{
+				Type:             "oauth2",
+				Flow:             "accessCode",
+				AuthorizationURL: config.OAuthConfig.MasterURL + "/oauth/authorize",
```

Comment:
Not sure I want to use `config.OAuthConfig.MasterURL` here because then it is easy for generated code to not line up between local env and Jenkins.